malsub/service/virustotal.py

```diff
@@ -28,9 +28,20 @@ class VirusTotal(Service):
     # https://www.virustotal.com/en/documentation/public-api/
     # https://www.virustotal.com/en/documentation/private-api/
 
-    @Service.unsupported
+    
     def download_file(self, hash: Hash):
-        pass
+        self.api_dowf.param = {**self.get_apikey(), "hash": hash.hash}
+        from requests.exceptions import HTTPError
+        try:
+            data, filename = request(self.api_dowf, bin=True)
+        except HTTPError as e:
+            if e.response.status_code == 404:
+                return f"sample \"{hash.hash}\" not found"
```

Comment:
You can replace this with `return f"sample \"{hash}\" not found"` in order to print the hash value and its algorithm as per the `__str__` definition of the `Hash` class.